### PR TITLE
[v8] Online release process with Github Actions

### PR DESCRIPTION
**Pull request by a-b**
The current state of CF CLI release infrastructure is offline and quite outdated. To make our release process more transparent and inclusive we decided to move to build, signing, and release processes online to Github Actions

### CODE REVIEW

**Review by reedr3 — APPROVED** [review]
Looks good to me

File: .github/win/innosetup/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
+SHELL ["powershell.exe"]
+
+ARG CHOCO_THUMBPRINT=83AC7D88C66CB8680BCE802E0F0F5C179722764B
+RUN mkdir \setup
+# RUN (New-Object System.Net.WebClient).DownloadFile('https://chocolatey.org/install.ps1', '\setup\installChocolatey.ps1'); \
```
Comment (jdgonzaleza) [inline .github/win/innosetup/Dockerfile:6]: Could we delete these lines?

Comment (jdgonzaleza) [inline .github/win/innosetup/Dockerfile:6]: Could we delete these lines?

**Review by jdgonzaleza — COMMENTED** [review]
LGTM, 
Maybe we could delete those commented/unused code

**Review by hjcalderon10 — APPROVED** [review]

**Review by a-b — COMMENTED** [review]
LGTM

**Comment by pivotalgeorge** [comment]
LGTM

**Pull request merged**